raddb/mods-available/eap

```diff
@@ -410,6 +420,12 @@ eap {
 		#
 		dh_file = ${certdir}/dh
 
+		#
+		#  disable_single_dh_use:: Always using it has an impact on the computer time needed
+		#  during negotiation, but it is not very large.
```

Comment:
That's not really a meaningful comment...

Reply:
@arr2036 updated